installer/pyinstaller/build-linux.sh

```diff
@@ -0,0 +1,86 @@
+binary_zip_filename=$1
```

Comment:
Should include a shebang line (e.g. `#!/bin/sh`) otherwise the chosen shell is not guaranteed when running `./build-linux.sh`.

Reply:
Added.